xla/stream_executor/cuda/cuda_dnn.h

```diff
@@ -311,6 +315,19 @@ class CudnnSupport : public dnn::DnnSupport {
       std::optional<dnn::TensorDescriptor> dscale_descriptor,
       std::optional<dnn::TensorDescriptor> dbias_descriptor) override;
 
+  // Queries whether a graph is generally supported.
+  absl::StatusOr<bool> PrepareGraph(
```

Comment:
With virtual graph base these APIs should become part of graph itself.

Reply:
Done.